repository/src/main/java/com/epam/esm/exception/InvalidPageContextException.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.exception;
+
+import com.epam.esm.ErrorType;
+
+public class InvalidPageContextException extends RuntimeException{
```

Comment:
The name looks rather ambiguous. Perhaps you can rename it to Pagination Exception?

Reply:
fixed